models/model-cards/gnn-fsi-model-card.md

```diff
@@ -0,0 +1,228 @@
+* None<br>
+
+## Output
+An anomalous score of transactions indicates a probability score of being a fraud.<br>
+**Output Parameters:**  
+* None <br>
+
+**Output Format:** 
+* CSV<br>
+
+**Other Properties Related to Output:** 
+* None <br> 
+
+## Software Integration:
+**Runtime(s):** 
+* Morpheus  <br>
+
+**Supported Hardware Platform(s):** <br>
+* Ampere/Turing <br>
+
+**Supported Operating System(s):** <br>
+* Linux <br>
+  
+## Model Version(s): 
+* 1.0 <br>
+
+### How To Use This Model
+This model is an example of a fraud detection pipeline using a graph neural network and gradient boosting trees. This can be further retrained or fine-tuned to be used for similar types of transaction networks with similar graph structures.
+
+# Training & Evaluation: 
+
+## Training Dataset:
+
+**Link:**
+* [fraud-detection-training-data.csv](models/dataset/fraud-detection-training-data.csv)  <br>
+
+**Properties (Quantity, Dataset Descriptions, Sensor(s)):** 
+* A training data consists of raw 753 synthetic labeled credit card transaction data with data augmentation in a total of 12053 labeled transaction data. <br>
+
+**Dataset License:**  
+* [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0) <br>
+
+## Evaluation Dataset:
+**Link:**  
+* [fraud-detection-validation-data.csv](models/dataset/fraud-detection-validation-data.csv)  <br>
+
+**Properties (Quantity, Dataset Descriptions, Sensor(s)):** 
+* Data consists of raw 265 labeled credit card transaction synthetically created<br>
+
+**Dataset License:**
+*  [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0)<br>
+
+## Inference:
+**Engine:** 
+* Triton <br>
+
+**Test Hardware:** <br>
+* Other   <br>
+
+# Subcards
+## Model Card ++ Bias Subcard
+
+### What is the gender balance of the model validation data?  
+* Not Applicable
+
+### What is the racial/ethnicity balance of the model validation data?
+* Not Applicable
+
+### What is the age balance of the model validation data?
+* Not Applicable
+
+### What is the language balance of the model validation data?
+* Not Applicable
+
+### What is the geographic origin language balance of the model validation data?
+* Not Applicable
+
+### What is the educational background balance of the model validation data?
+* Not Applicable
+
+### What is the accent balance of the model validation data?
+* Not Applicable
+### Describe measures taken to mitigate against unwanted bias.
+* Not Applicable
+## Model Card ++ Explainability Subcard
+
+### Name example applications and use cases for this model. 
+* This model is intended to be used for testing fraud detection use cases in morpheus pipeline using application of GNN.
+### Fill in the blank for the model technique.
+* This model is intended for developers that want to build and/or customize fraud detection applications using graph neural network.
+
+### Name who is intended to benefit from this model. 
+* This model is intended for users that use transactional data to build fraud detection application.
```

Comment:
Same as above, let's focus on testing.
Example answers for DFP:
`The intended beneficiaries of this model are developers who aim to test the performance and functionality of the DFP pipeline using synthetic datasets. It may not be suitable or provide significant value for real-world cloudtrail logs analysis.`